features/bookmarks/bookmark_browse.feature

```diff
@@ -16,3 +15,11 @@ Feature: Browse Bookmarks
     When I go to the bookmarks page for user "ethel" with pseud "aka"
     Then I should see "Bookmarked with Other Pseud"
       And I should not see "Bookmarked with Default Pseud"
+
+  Scenario: When logged out, the "save" button on bookmarks does not show
+    Given I am logged in as "bookmarker"
+      And I bookmark the work "Test"
+    When I log out
+      And I go to the bookmarks page for the user "bookmarker"
```

Comment:
This is the wrong page to check - it doesn't have the bug that this is fixing. The test should visit the bookmarks page for a tag instead. Looking at the previous test failure, I think the issue with visiting the tags bookmarks page/making the bookmark was that the tag didn't exist yet.